app/serializers/api/admin/for_order_cycle/enterprise_serializer.rb

```diff
@@ -28,15 +28,20 @@ def supplied_products
 
   private
 
-  def products
-    return @products unless @products.nil?
-    @products = if order_cycle.prefers_product_selection_from_coordinator_inventory_only?
+  def products_scope
+    if order_cycle.prefers_product_selection_from_coordinator_inventory_only?
       object.supplied_products.visible_for(order_cycle.coordinator)
     else
       object.supplied_products
     end
   end
 
+  def products
+    return @products unless @products.nil?
+
+    @products = products_scope.includes(:supplier, master: [:images])
```

Comment:
Can lines 40-42 just be replaced with this?
```
@products ||= products_scope.includes(:supplier, master: [:images])
```

Reply:
Oh, right! I didn't notice, I was updating just the bottom part of the method. Code simplified here: https://github.com/openfoodfoundation/openfoodnetwork/pull/3802/commits/56bc497fe272e1549c71228f56da3519044536c8 @luisramos0